test/betterbench.cpp

```
@@ -0,0 +1,41 @@
+    std::cout << __FILE__ << std::endl;
+
+    base_type b0 = 0, b1 = 0, b2 = 0, b3 = 0;
+    op(b0, b1, b2, b3);
+    KLOG(INF) << b3;
+
+    {
+        SoA<MKN_AVX_BENCH_TYPE, base_type> soa{size};
+        assert(soa);
+        soa();
+        if (soa.v3.data()[soa.v3.size() - 1] != b3)
+            throw std::runtime_error("FAIL!");
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)